### PR TITLE
Add flask-mqtt

### PR DESCRIPTION
**Pull request by stlehmann**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there (I am the maintainer)

### CODE REVIEW

File: recipes/flask-mqtt/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  version: {{ version }}
+
+source:
+  url: https://github.com/stlehmann/Flask-MQTT/releases/download/{{ version }}/Flask-MQTT-{{ version }}.tar.gz
+  sha256: d196c4a7a1fc37a4364e8fe46b3670cc656a851c5b63d69e2836c80e03da88f1
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<36]
```
Comment (npavlovikj) [inline recipes/flask-mqtt/meta.yaml]: I think this recipe can be `noarch`, https://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages. Please remove the `skip` line and add `noarch: python` in that case.

File: recipes/flask-mqtt/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+requirements:
+  build:
```
Comment (npavlovikj) [inline recipes/flask-mqtt/meta.yaml]: You can delete `build`.

File: recipes/flask-mqtt/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  host:
```
Comment (npavlovikj) [inline recipes/flask-mqtt/meta.yaml]: In general, only `pip` and `python` are enough for the `host` section. Can you please try that?

File: recipes/flask-mqtt/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    - python
```
Comment (npavlovikj) [inline recipes/flask-mqtt/meta.yaml]: Use `python >=3.6`.

File: recipes/flask-mqtt/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+    - pip
+    - flask
+    - paho-mqtt
+  run:
+    - python
```
Comment (npavlovikj) [inline recipes/flask-mqtt/meta.yaml]: Use `python >=3.6`.

**Comment by stlehmann** [comment]
@npavlovikj Thanks for your help. I tried to fix all the issues. Somehow the AppVeyor build continues failing. Do you have any idea about that?

**Comment by npavlovikj** [comment]
`noarch` recipes are only built on Linux, so any other failure is ok.

**Comment by npavlovikj** [comment]
Btw, the recipe looks good to me, so once the tests pass, we can go ahead and merge it.

**Comment by stlehmann** [comment]
Great

**Comment by npavlovikj** [comment]
Thank you for contributing @stlehmann 🎉

**Pull request merged**